src/battle_main.c

```diff
@@ -166,8 +167,8 @@ EWRAM_DATA static u32 sUnusedUnknownArray[25] = {0};
 EWRAM_DATA u32 gBattleTypeFlags = 0;
 EWRAM_DATA u8 gBattleTerrain = 0;
 EWRAM_DATA u32 gUnknown_02022FF4 = 0;
-EWRAM_DATA struct UnknownPokemonStruct4 gUnknown_02022FF8[3] = {0}; // what is it used for?
-EWRAM_DATA struct UnknownPokemonStruct4* gUnknown_02023058 = NULL; // what is it used for?
+EWRAM_DATA struct UnknownPokemonStruct4 gMultiPartnerParty[PARTY_SIZE / 2] = {0};
```

Comment:
I'd rather see a define for `MULTI_PARTY_SIZE = PARTY_SIZE / 2`, rather than see `PARTY / 2` everywhere in the code.

Reply:
Sounds good. It gets sort of messy with the size of Battler Frontier parties, which are also 3 despite not being multi parties, because they both use the same "choose 3 mons from the party menu" logic. I defined both constants (`MULTI_PARTY_SIZE` and `FRONTIER_PARTY_SIZE`) for now, but if someone wanted to change the size of you'd have to handle them differently.

Reply:
Also I added one instance of `PARTY_SIZE / 2` for `gBattlePartyCurrentOrder` because it literally is the party packed into 4 bits at a time